crates/bevy_android/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#[cfg(target_os = "android")]
+use bevy_ecs::system::Resource;
+
+#[cfg(target_os = "android")]
+pub use ndk::asset::AssetManager;
+#[cfg(target_os = "android")]
+pub use winit::platform::android::activity::*;
+
+/// Resource containing AndroidApp
+#[cfg(target_os = "android")]
+pub struct AndroidResource {
+    pub android_app: AndroidApp
+}
+
+#[cfg(target_os = "android")]
+impl Resource for AndroidResource {}
```

Comment:
I don't think this justify creating a new crate for it. Also, I would rather avoid having a `bevy_android` crate where everything is behind a check for android.

Could it be placed in an existing crate?